contributing_to_docs/doc_guidelines.adoc

```diff
@@ -911,22 +911,134 @@ kind: Pod
 metadata:
   labels:
     test: liveness
-...
+# ...
 ----
 +
 or
 +
 ----
-...
+# ...
 spec:
   containers:
   - args:
     image: k8s.gcr.io/liveness
-...
+# ...
 ----
 +
 Do not use `[...]`, `<snip>`, or any other variant.
 
+* Do not use `[source,yaml]` for the following:
```

Comment:
Are users going to be confused that it's saying not to use `[source,yaml]` but then the examples show `terminal` and `text` syntax? Not that I have a better suggestion. 😀  Maybe you could say something like, "Do not use `[source,yaml]` when describing terminal commands or text output for the following types"?